### ISSUE TITLE
The loss didn't converge to zero during fine-tunning in my own data.

### ISSUE THREAD

**Issue by kephakhang** [fferroni/DEC-Keras#5]
DEC-Keras network is so wonderful for my  data.

I changed the input as my data instead of mnist data.

The pre_trainning is so good.

But the loss value diverge to NaN  during fine tunning.

I think that dynamic learning rate will be needed in fine tunning !

Could you suggest me the best method for applying dynamic learning rate.

Thank you in advance.

**Comment by fferroni** [comment]
You can change the learning rate schedule, and/or clip your gradients, and/or use a different optimiser other than SGD.
